src/App.vue

```diff
@@ -65,6 +65,23 @@
             <li>Try different keywords.</li>
           </ul>
         </div>
+        <footer>
+          <div class="developer-cta">
+            <small>
+              <strong>Are you a DApp builder?</strong>
+              <br />
+              <a href="https://docs.vechain.org/developer-resources/getting-started" target="_blank">
+                Read the docs
+              </a> 
+              and
+              <a href="https://github.com/vechain/app-hub#vechain-app-hub---submit-form" target="_blank">
+                submit your DApp
+              </a>
+            </small>
+          </div>
+          <div class="divider"></div>
+          <small>All Rights Reserved. © 2024 Vechain Foundation San Marino S.r.l.</small>
```

Comment:
How about making the year dynamic? Should it be `2018-2024` or just `2024`? And `VeChain foundation`?

Reply:
The footer have been copied by vechain.org... strange...  what would be the correct one? @libotony 
I've made the year dynamic.

Reply:
Let's keep aligned with the org website.